### DISCUSSION TITLE
[Feat request]: Elasticsearch annotations & support for nested timestamp field

### DISCUSSION THREAD

**Discussion by ajrnz** [grafana/grafana#56261] [Observability]
Elasticsearch based annotations don't show up if the time field is a nested field. For example: `entry.date`

The query from the server is retrieving the correctly and the data is present however my guess is that the date field is not being dereferenced properly. If you are using the `fields` field then `x.fields.entry.date` would be incorrect as you would need something more like `x.fields.'entry.date'`

**Comment by gabor** [comment]
hi @ajrnz , sorry for the late reply, is this problem still happening? if yes, could you describe it in more detail?
could you maybe provide a sample elasticsearch object that demonstrates the problem?

**Comment by ajrnz** [comment]
I'm not sure. In the end we had to change out data model to bring the timestamp to the top level of the object. 
I don't have much to do with this any more so you can close it from my point of view.
